WcaOnRails/app/views/shared/_wca_id.html.erb

```diff
@@ -2,5 +2,6 @@
 <%# We use a full url because this can be rendered inside of emails -%>
 <% options = {} %>
 <% options[:target] = target if defined?(target) %>
-<%= link_to wca_id, "#{root_url}results/p.php?i=#{wca_id}", options -%>
+<% # FIXME restore single 'root_url' when (if) results is on Rails. %>
+<%= link_to wca_id, "#{root_url(locale:nil)}results/p.php?i=#{wca_id}", options -%>
```

Comment:
I don't know :)

In both alternatives we have to remember to change something once the results page is ported to rails. I like "my" alternative because it involves changing one line in a file, whereas the other implies learning how to change the nginx thing, and (probably) changing one line in a file.

It's your call :p